storage/Utils/LockImpl.cc

```diff
@@ -66,8 +66,10 @@ namespace storage
 		y2err("creating directory for lock-file failed: " << strerror(errno));
 	    }
 
-	    fd = open(LOCK_DIR "/lock", (read_only ? O_RDONLY : O_WRONLY) | O_CREAT | O_CLOEXEC,
-		      S_IRUSR | S_IWUSR | S_IRGRP | S_IWGRP | S_IROTH | S_IWOTH);
+	    const int permissions = S_IRUSR | S_IWUSR | S_IRGRP | S_IROTH; // 0644
```

Comment:
Personally I find the octal value 0644 **much** better to read. Why use those constants?

Reply:
In fact, I had it with octal numbers, but not sure if octal value and constants will remain the same in the future. So, I came with the "safest" version.

Reply:
If this ever changes, we are doomed.

Unix permissions have relied on this since the early 1970. This is not anything that will possibly _ever_ change.

Seriously, **nobody** uses those constants, and with the cryptic letters in them, they are a lot harder to read than the octal numbers.

Reply:
Ok, I will change them.